components/inputs/test/input-color.visual-diff.js

```diff
@@ -13,6 +13,10 @@ describe('d2l-input-color', () => {
 		await page.bringToFront();
 	});
 
+	beforeEach(async() => {
+		await resetFocus(page);
+	});
```

Comment:
Was getting a lot of flake in these input-color tests... not totally sure why it's cropping up now, but resetting the focus in between tests and waiting a bit longer for the tooltip (like we do in other tests) seemed to fix it consistently.